components/button/style.scss

```diff
@@ -41,6 +44,15 @@
 	.wp-core-ui.gutenberg-editor-page & {
 		font-size: $default-font-size;
 	}
+
+	&.external-link {
+		.dashicon {
+			width: 1.4em;
```

Comment:
I see these existed previously, but shouldn't these styles be inherited from `IconButton` ? It's not obvious to me why we would we want the external link icon to be displayed any differently than any other icon button?